cardano-api/src/Cardano/Api/MetaData.hs

```diff
@@ -123,49 +129,41 @@ jsonToMetadataValue  av =
         case Scientific.floatingOrInteger sci :: Either Double Integer of
           Left  n -> Left (ConversionErrNumberNotInteger n)
           Right n -> Right (Api.TxMetaNumber n)
-      Aeson.String txt -> jsonToMetadataString txt
+      Aeson.String txt ->
+        case Text.stripPrefix bytesPrefix txt of
+          Nothing  -> jsonToMetadataString txt
+          Just hex -> case Base16.decode (Text.encodeUtf8 hex) of
+            (bytes, "") -> jsonToMetadataBytes bytes
+            (_, err)    -> Left $ ConversionErrBadBytes err
```

Comment:
Seems like this kind of check should be done by a separate linter than be rejected here.

Reply:
@intricate yes it is fine. Metadata aren't meant to be a place of self expression but rather a compact way to embed custom information in transactions. So rather than free-form text and verbose sentence, one should put the minimal information possible in metadata and have more verbose details stored off chain.